markdown/usage/installation.md

```diff
@@ -31,125 +18,52 @@ java -version
-
-Then, running the pipeline with the option `-profile docker` tells Nextflow to enable Docker for this run.
-An image containing all of the software requirements will be automatically fetched and used from dockerhub
-([https://hub.docker.com/r/nfcore/YOUR_PIPELINE](https://hub.docker.com/r/nfcore/YOUR_PIPELINE)).
-
-### Singularity
-If you're not able to use Docker then [Singularity](http://singularity.lbl.gov/) is a great alternative.
-The process is very similar: running the pipeline with the option `-profile singularity` tells Nextflow to enable singularity for this run. An image containing all of the software requirements will be automatically fetched and used from singularity hub.
-
-If running offline with Singularity, you'll need to download and transfer the Singularity image first:
-
-```bash
-singularity pull --name nf-core/YOUR_PIPELINE_NAME-1.0.img docker://nfcore/YOUR_PIPELINE_NAME:1.0
-```
-
-Once transferred, use `-with-singularity` and specify the path to the image file:
+### Automatic
 
-```bash
-nextflow run /path/to/nf-core-YOUR_PIPELINE -with-singularity nf-core-YOUR_PIPELINE.simg
-```
+This pipeline itself needs no installation - nextflow will automatically fetch it from GitHub if `nf-core/<pipeline-name>` is specified as the pipeline name.
 
-Remember to pull updated versions of the singularity image if you update the pipeline.
+This method requires an internet connection. If you're running on a system that has no internet connection, please see the [Running Offline](/usage/offline) documentation.
 
-### Conda
-If you're not able to use Docker _or_ Singularity, you can instead use conda to manage the software requirements.
-This is slower and less reproducible than the above, but is still better than having to install all requirements yourself!
-The pipeline ships with a conda environment file and nextflow has built-in support for this.
-To use it first ensure that you have conda installed (we recommend [miniconda](https://conda.io/miniconda.html)), then follow the same pattern as above and use the flag `-profile conda`
+### Development
 
-### Configuration profiles
+If you would like to make changes to the pipeline, fork the GitHub repository and then clone the files. Once cloned you can run the pipeline with `nextflow run <path-to-repo>`.
 
-See [Adding your own configuration profile](adding_own_config).
+Note that you should _only_ do this if you intend to make significant changes to the pipeline. All configuration options can be changed without editing the pipeline code. Forking the pipeline repositories means that you cannot use stable releases and you will fall behind new updates.
 
 ## Reference genomes
 
-See [Reference genomes](reference_genomes)
+Some pipelines come with built-in support for iGenomes references. It may be preferable for you to download a local copy of these to your system. For more information, see [_Reference genomes_](reference_genomes).
```

Comment:
Maybe mention its better to download locally to overcome download issues is using the same reference  and to save storage space.